website/content/docs/platform/k8s/csi/index.mdx

```diff
@@ -0,0 +1,130 @@
+---
+layout: docs
+page_title: Vault CSI Provider
+sidebar_title: Vault CSI Provider
+description: >-
+  The Vault CSI Provider allows pods to consume Vault secrets using CSI Secret Store
+  volumes.
+---
+
+# Vault CSI Provider
+
+The Vault CSI Provider allows pods to consume Vault secrets using 
+[CSI Secret Store](https://github.com/kubernetes-sigs/secrets-store-csi-driver) volumes.
+
+~> The Vault CSI Provider requires the [CSI Secret Store](https://github.com/kubernetes-sigs/secrets-store-csi-driver) 
+  Driver to be installed.
+
+## Overview
+
+At a high level, the CSI Secret Store controller allows users to create `SecretStoreClass` objects. 
+This object defines which secret provider to use and what secrets to retrieve. When the provider 
+used is `vault`, the Vault CSI Provider will receive the request, use the pods service account to retrieve 
+the specified secrets from Vault, and share them with the pod via a host path volume.
```

Comment:
```suggestion
the specified secrets from Vault, and mount them into the pod's CSI volume.
```

The host path volume is shared with the driver, not the requesting pod, and the secrets will probably soon be sent back to the driver over gRPC instead of written directly to the host path volume. So I think a small tweak here makes it a bit more accurate and saves us needing to update it again in the near future.